src/main/java/jbash/commands/CmdCd.java

```diff
@@ -0,0 +1,29 @@
+package jbash.commands;
+
+import jbash.filesystem.FileSystemAPI;
+
+import java.util.List;
+
+public class CmdCd extends Command {
+    CmdCd(String name) {
+        super("cd");
+    }
+
+    @Override
+    public String getHelp() {
+        return "Usage: cd [directory]";
+    }
+
+    @Override
+    public int execute(List<String> args) {
+        if (args.size() > 1) { System.out.println("cd: too many arguments"); return 1; }
```

Comment:
This reminded me -- we gotta do this #6

Reply:
Yeah honestly that would be better than this nonsense